src/Catalyst.Modules.POA.P2P.Discovery.Consortium/Discovery/PoaDiscovery.cs

```diff
@@ -51,7 +51,7 @@ public PoaDiscovery(IPeerRepository peerRepository, IFileSystem fileSystem, ILog
         }
 
         /// <summary>
-        ///     @TODO get from container eventually 
+        ///     @TODO get from container eventually
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)